### DISCUSSION TITLE
Map a range of two values to one value for comparison

### DISCUSSION THREAD

**Discussion by gregorsart** [neuefische/questions#761] [Web]
Link to our repo: [Dog App](https://github.com/alinarub/dog-app/tree/api-test)

In our app we only have three answer options, but we have a corresponding rating scale given by the api from 1 to 5.
We want to assign it as follows.

The API looks like this: 

```js
const dummyDogs = [
  {
    trainability: 5,
    energy: 3,
    barking: 1,
    name: "Golden Retriever",
  },
  {
    trainability: 1,
    energy: 2,
    barking: 5,
    name: "Dachshund",
  },
  {
    trainability: 2,
    energy: 4,
    barking: 3,
    name: "German Shepherd",
  },
  {
    trainability: 5,
    energy: 5,
    barking: 2,
    name: "Border Collie",
  },
  {
    trainability: 3,
    energy: 2,
    barking: 3,
    name: "Pug",
  },
];
```

If the user clicks answer one (of three options), the ratings 1 and 2 from the api will be combined and so the results for 1 and 2 are shown. Respectively answer three will show the combined results of 4 and 5 from the api and answer two will show 3 from the api.

![Bildschirmfoto 2023-10-16 um 12 08 08](https://github.com/neuefische/questions/assets/32411569/5dd4a76c-460c-455c-8320-4ce4f1bac73d)

Ty in advance.

**Comment by NikRoe** [comment]
We talked about it and came to the solution of starting with 3 options and only fetching dogs with values of 2-4
